include/seqan3/alphabet/quality/concept.hpp

```diff
@@ -85,12 +85,12 @@ template <typename alphabet_type>
 //! public setter function receiving char encoding of phred score
 template <typename alphabet_type>
     requires detail::internal_quality_concept<alphabet_type>
-    constexpr alphabet_type from_phred(alphabet_type & c, char const in)
+    constexpr alphabet_type assign_phred(alphabet_type & c, char const in)
```

Comment:
they are functions, not meta functions... Of course we need them :)
Yes, it should be `&` on returning, but the entire quality submodule is WIP by @mariehoffmann (see #39) and I don't want to edit everything there now. If we get the rest of the alphabet module into a good state, I can then explain the design decisions to @mariehoffmann so that the quality submodule an be adapted in a similar manner (e.g. remove the internal_alphabet ..).